functions/components/security/network.js

```diff
@@ -207,7 +206,6 @@ authRouter.post('/refresh', (req, res) => {
             } else {
                 getOrganizationById(claims.organization)
```

Comment:
This kind of situations is where the getOrganizationById consumers must request the clean object without hydration, which includes properties and methods that corresponds to a Mongoose Object. Could we please correct this to work with the solution provided above?